src/app/(home)/_components/HomeFooter.tsx

```
@@ -0,0 +1,38 @@
+  margin: 0 auto;
+  a {
+    text-decoration: none;
+  }
+`;
+
+function HomeFooter() {
+  return (
+    <FooterWrapper className="p-medium-60 flex justify-between align-center g-medium-60">
+      <p className="fs-medium-10">
+        <span className="opacity-default-60">Built by</span>&nbsp;
+        <a target="_blank" href="https://github.com/nnsncl">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)